src/main.ts

```diff
@@ -1,10 +1,16 @@
 // Startup the audit service
+import { RabbitEventBus } from '@libero/event-bus';
 import { InfraLogger as logger } from './logger';
-import setup from './setup';
-import config from 'config';
-
-setup(config).then(app =>
-    app.listen(config.port, () => {
-        logger.info(`Audit service listening on port ${config.port}`);
-    }),
-);
+import { serviceConfig } from './config';
+import App from './app';
+
+const main = async (): Promise<void> => {
```

Comment:
Can we export this function and add a unit test to ensure its calling the functions expected and passing the return of `serviceConfig` to the `App` constructor.

Reply:
This was why we had the startup code extracted to the `setup.ts` file as the execution on import is a pain for unit testing.